config_gen.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
```

Comment:
This means we need to support both Python 2 and Python 3, because what `python` points to varies with distro/OS. I tried doing that in the past and found it quite frustrating, but given you're setting up CI at the same time that seems like a reasonable solution.